sidebars.js

```diff
@@ -334,6 +339,19 @@ const sidebars = {
           id: "protocol/ISM/wormhole-ISM",
           label: "Wormhole ISM",
         },
+        {
+          type: "category",
+          label: "Third-party ISMs",
```

Comment:
Is this creating a new category for just this one? seems like the public version has the other ISMs in the third party category too https://docs.hyperlane.xyz/docs/protocol/ISM/modular-security, just wanna make sure we keep this (or do away with the category)

Reply:
I will rebase off main